core/src/main/java/org/frankframework/align/XmlTypeToJsonSchemaConverter.java

```diff
@@ -485,20 +485,15 @@ private void handleElementDeclaration(JsonObjectBuilder builder, XSElementDeclar
 	// Currently commented out because builder param isnt used
 	// private void buildWildcard(JsonObjectBuilder builder, XSTerm term){
 	private void handleWildcard(XSWildcard wildcard){
-		String processContents;
 		switch (wildcard.getProcessContents()) {
-		case XSWildcard.PC_LAX: processContents="LAX"; break;
-		case XSWildcard.PC_SKIP: processContents="SKIP"; break;
-		case XSWildcard.PC_STRICT: processContents="STRICT"; break;
-		default:
+			case XSWildcard.PC_LAX, XSWildcard.PC_SKIP, XSWildcard.PC_STRICT: break;
+			default:
 				throw new IllegalStateException("handleWildcard wildcard.processContents is not PC_LAX, PC_SKIP or PC_STRICT, but ["+wildcard.getProcessContents()+"]");
 		}
-		String namespaceConstraint;
+
 		switch (wildcard.getConstraintType()) {
-		case XSWildcard.NSCONSTRAINT_ANY : namespaceConstraint="ANY"; break;
-		case XSWildcard.NSCONSTRAINT_LIST : namespaceConstraint="SKIP "+wildcard.getNsConstraintList(); break;
-		case XSWildcard.NSCONSTRAINT_NOT : namespaceConstraint="NOT "+wildcard.getNsConstraintList(); break;
-		default:
+			case XSWildcard.NSCONSTRAINT_ANY, XSWildcard.NSCONSTRAINT_LIST, XSWildcard.NSCONSTRAINT_NOT : break;
+			default:
 				throw new IllegalStateException("handleWildcard wildcard.namespaceConstraint is not ANY, LIST or NOT, but ["+wildcard.getConstraintType()+"]");
 		}
 	}
```

Comment:
Als ik in intellij de suggesties volg (switch met single case -> if, invert lege if), kom ik op het volgende uit, wat IMO wat beter wegleest:

```
	private void handleWildcard(XSWildcard wildcard){
		short processContents = wildcard.getProcessContents();
		if (processContents != XSWildcard.PC_LAX && processContents != XSWildcard.PC_SKIP && processContents != XSWildcard.PC_STRICT) {
			throw new IllegalStateException("handleWildcard wildcard.processContents is not PC_LAX, PC_SKIP or PC_STRICT, but [" + wildcard.getProcessContents() + "]");
		}

		short constraintType = wildcard.getConstraintType();
		if (constraintType != XSWildcard.NSCONSTRAINT_ANY && constraintType != XSWildcard.NSCONSTRAINT_LIST && constraintType != XSWildcard.NSCONSTRAINT_NOT) {
			throw new IllegalStateException("handleWildcard wildcard.namespaceConstraint is not ANY, LIST or NOT, but [" + wildcard.getConstraintType() + "]");
		}
	}
```